src/incubator/f5-bigip-ctlr/Chart.yaml

```
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: Deploy the F5 Networks BIG-IP Controller for Kubernetes and OpenShift (k8s-bigip-ctlr).
 name: f5-bigip-ctlr
-version: 0.0.4
+version: 0.0.5
```

Version should be updated to `0.0.7`, as the current version is `0.0.6`.

```suggestion
version: 0.0.7
```